src/lib/web-server.js

```diff
@@ -49,6 +50,19 @@ web_server_app.get('/normalised_data/all', async (req, res) => {
     res.json(out);
 });
 
+
+web_server_app.get('/normalised_data/item_by_id/:data_id', async (req, res) => {
```

Comment:
Minor thing (not blocking) but can we drop "_by_id" that feels a little cumbersome. "/normalised_data/item/:data_id" looks better.